raspimouse_navigation_examples/raspimouse_navigation_examples/waypoint.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+# coding: UTF-8
+
+# Copyright 2024 RT Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+
+from geometry_msgs.msg import PoseStamped
+from nav2_simple_commander.robot_navigator import BasicNavigator, TaskResult
+
+import rclpy
+from rclpy.duration import Duration
+
+
+def generate_pose(navigator, x: float, y: float, deg: float) -> PoseStamped:
+    wp = PoseStamped()
+    wp.header.frame_id = 'map'
+    wp.header.stamp = navigator.get_clock().now().to_msg()
+    wp.pose.position.x = x
+    wp.pose.position.y = y
```

Comment:
```suggestion
    wp.pose.position.x = x
    wp.pose.position.y = y
    wp.pose.position.z = 0.0
```

個人的な好みになりますが、z=0.0も明示したほうがサンプルとして分かりやすいかもしれません。